### PR TITLE
Support ignore option in function_naming_convention

### PR DESCRIPTION
**Pull request by define-null**

### CODE REVIEW

**Comment by define-null** [comment]
ping

File: test/style_SUITE.erl [hunk 1]
```diff
@@ -92,7 +92,15 @@ verify_function_naming_convention(_Config) ->
     {ok, FileFail} = elvis_test_utils:find_file(SrcDirs, PathFail),
     [_CamelCaseError, _ALL_CAPSError, _InitialCapError,
      _HyphenError, _PredError, _EmailError] =
-        elvis_style:function_naming_convention(ElvisConfig, FileFail, RuleConfig).
+      elvis_style:function_naming_convention(ElvisConfig, FileFail, RuleConfig),
+
+    RuleConfig2 = #{regex => "^([a-z][a-z0-9]*_?)*$",
+                    ignore => [fail_function_naming_convention]
+                   },
+    PathFail = "fail_function_naming_convention.erl",
```
Comment (harenson) [inline test/style_SUITE.erl]: Please, remove this line, this variable was already declared on line 91.

**Comment by define-null** [comment]
@harenson fixed

**Comment by harenson** [comment]
Thanks :+1:

**Pull request merged**